### PR TITLE
Mainnet node release April 2023

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform users about changes in Mainnet node release.

## Changes

Release notes, downloads, update to Ubuntu node file, previous downloads

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

**Review by abizjak — APPROVED** [review]
Looks good.

File: source/mainnet/net/guides/run-node-ubuntu.rst [hunk 1]
```diff
@@ -104,6 +104,27 @@ If the node is well behind the head of the chain, you can speed up the startup b
 
 After the node is caught up remove the out of band catchup configuration to speed up further node restarts.
 
+.. _node-collector-ubuntu-mainnet:
+
+Node collector configuration
+============================
+
+Since version 5.3.0 of the node the collector uses the GRPC V2 interface. Therefore, in order to run the collector, it is required that the node which the collector connects to has the GRPC V2 interface enabled.
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/guides/run-node-ubuntu.rst:112]: Yes, just forgot to update it from the original.

**Review by mh-concordium — CHANGES_REQUESTED** [review]
Looks good, I only noticed a minor thing.

File: source/mainnet/net/guides/run-node-ubuntu.rst [hunk 2]
```diff
@@ -104,6 +104,27 @@ If the node is well behind the head of the chain, you can speed up the startup b
 
 After the node is caught up remove the out of band catchup configuration to speed up further node restarts.
 
+.. _node-collector-ubuntu-mainnet:
+
+Node collector configuration
+============================
+
+Since version 5.3.x of the node the collector uses the GRPC V2 interface. Therefore, in order to run the collector, it is required that the node which the collector connects to has the GRPC V2 interface enabled.
```
[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/guides/run-node-ubuntu.rst:112]: This x was what Ales requested in the PRs for Testnet release. So I did the same for mainnet Ubuntu for this release.

[the remark this answers is not in the document]
Comment (dg-concordium) [reply source/mainnet/net/guides/run-node-ubuntu.rst:112]: https://github.com/Concordium/concordium.github.io/pull/783/files

**Review by mh-concordium — APPROVED** [review]
Looks good.

**Pull request merged**